src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -656,6 +662,7 @@ const FileField = props => {
           {(maxItems === null || files.length < maxItems) &&
             // Prevent additional upload if any upload has error state
             checkUploadVisibility() && (
+              // eslint-disable-next-line jsx-a11y/label-has-associated-control
```

Comment:
This pattern is using a dynamically rendered label with nested button; with an always rendered hidden file input after the label. It hasn't been updated to use the file web component because of the difficulty in updating the unit tests. I added a comment to disable the eslint warning for the label.